File: src/asp_net_backend/src/api/Models/RoomType.cs
namespace Restify.API.Models;

public class RoomType
{
	public decimal	Fees { get; set; }
	public decimal	PerNightPrice { get; set; }
	public uint		Capacity { get; set; }
    public uint     Id { get; set; }
    public uint     LodgingId { get; set; }
    public string   Name { get; set; }
    
    public IList<RoomTypePhoto>	Photos { get; }

    public RoomType()
    {
	    Photos = new List<RoomTypePhoto>(0);
    }
}

File: src/frontend/src/app/models/room_type.ts
export class RoomType
{
    public constructor(
        public id:              number,
        public fees:            number,
        public perNightPrice:   number,
        public capacity:        number,
        public lodgingId:       number,
        public name:            string,
        public photos:          string[]
    ) { }
}